cove/management/commands/base_command.py

```diff
@@ -19,28 +19,37 @@ def __init__(self, *args, **kwargs):
         super(CoveBaseCommand, self).__init__(*args, **kwargs)
 
     def add_arguments(self, parser):
-        parser.add_argument('file', help='File to be processed by Cove')
+        parser.add_argument('files', help='Files to be processed by Cove', nargs='+')
         parser.add_argument('--output-dir', '-o', default='', help='Directory where the output is created, defaults to the name of the file')
         parser.add_argument('--delete', '-d', action='store_true', help='Delete existing directory if it exits')
         parser.add_argument('--exclude-file', '-e', action='store_true', help='Do not include the file in the output directory')
+        parser.add_argument('--stream', '-t', action='store_true', help='The output would be streamed to stdout')
+
+    def handle(self, files, *args, **options):
+
+        stream = options.get('stream')
+        if (len(files) > 1 or os.path.isdir(files[0])) and not stream:
+            self.stdout.write('The multiple file option must be used with the --stream option')
```

Comment:
I think we need for streaming to print every error out to stderr not stdout.  This is because the user will expect the stdout to be in JSON lines format.  If we print out anything non JSON to stdout then it will cause the user an issue.